### PR TITLE
Change the text color

### PR DESCRIPTION
**Pull request by Shivanshu2003**
Targets #2324

### Description
Changed the background color of delete button

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: resources/views/knowledgecafe/library/categories/index.blade.php [hunk 1]
```diff
@@ -62,7 +62,7 @@ class ="table-bordered">
                    </button>
                 </div>
                 
-                <div class="text-danger c-pointer ml-3" @click="deleteCategory(index)">
+                <div  class="btn btn-danger" @click="deleteCategory(index)">
```
Comment (gautam071) [inline resources/views/knowledgecafe/library/categories/index.blade.php:65]:
```suggestion
                <div class="btn btn-danger" @click="deleteCategory(index)">
```

**Review by gautam071 — CHANGES_REQUESTED** [review]

File: resources/views/knowledgecafe/library/categories/index.blade.php (hunk 1 above)
Comment (Ayush8923) [inline resources/views/knowledgecafe/library/categories/index.blade.php:65]: Why removed the `c-pointer` and `ml-3` class?

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comment.

File: resources/views/knowledgecafe/library/categories/index.blade.php [hunk 2]
```diff
@@ -62,7 +62,7 @@ class ="table-bordered">
                    </button>
                 </div>
                 
-                <div class="text-danger c-pointer ml-3" @click="deleteCategory(index)">
+                <div  class="btn btn-danger" class="text-danger c-pointer ml-3" @click="deleteCategory(index)">
```
Comment (Ayush8923) [inline resources/views/knowledgecafe/library/categories/index.blade.php:65]: Why removed the `c-pointer` and `ml-3` class?

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comment.

File: resources/views/knowledgecafe/library/categories/index.blade.php [hunk 3]
```diff
@@ -62,7 +62,7 @@ class ="table-bordered">
                    </button>
                 </div>
                 
-                <div class="text-danger c-pointer ml-3" @click="deleteCategory(index)">
+                <div  class="btn btn-danger text-white c-pointer ml-3"  @click="deleteCategory(index)">
```
Comment (Ayush8923) [inline resources/views/knowledgecafe/library/categories/index.blade.php:65]: @Shivanshu2003 Remove the extra 2 spaces used before class and after class.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comment.

**Review by Ayush8923 — APPROVED** [review]
Looks good.

**Comment by Ayush8923** [comment]
@gautam071 Can you take a look at this PR once?

**Review by Ayush8923 — APPROVED** [review]

**Review by Ayush8923 — APPROVED** [review]

**Review by gautam071 — APPROVED** [review]

**Pull request merged**